resteasy-spring-boot-starter/pom.xml

```diff
@@ -316,329 +317,336 @@
+        <!--                            <id>sca-translate-exec</id>-->
+        <!--                            <goals>-->
+        <!--                                <goal>translate</goal>-->
+        <!--                            </goals>-->
+        <!--                        </execution>-->
+        <!--                    </executions>-->
+        <!--                    <version>4.22</version>-->
+        <!--                </plugin>-->
+        <!--                &lt;!&ndash; End Added For Fortify Support &ndash;&gt;-->
+        <!--            </plugins>-->
+        <!--        </pluginManagement>-->
         <plugins>
-
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <argLine>${modular.jdk.args} ${modular.jdk.props}</argLine>
+                    <argLine>${modular.jdk.args} ${modular.jdk.props} --add-opens java.base/java.lang=ALL-UNNAMED
+                        -Dillegal-access=permit
+                    </argLine>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>enforce</id>
-                        <configuration>
-                            <rules>
-                                <DependencyConvergence/>
-                            </rules>
-                        </configuration>
-                        <goals>
-                            <goal>enforce</goal>
-                        </goals>
-                        <phase>validate</phase>
-                    </execution>
-                </executions>
-            </plugin>
+            <!--            <plugin>-->
+            <!--                <groupId>org.apache.maven.plugins</groupId>-->
+            <!--                <artifactId>maven-enforcer-plugin</artifactId>-->
+            <!--                <executions>-->
+            <!--                    <execution>-->
+            <!--                        <id>enforce</id>-->
+            <!--                        <configuration>-->
+            <!--                            <rules>-->
+            <!--                                <DependencyConvergence/>-->
+            <!--                            </rules>-->
+            <!--                        </configuration>-->
+            <!--                        <goals>-->
+            <!--                            <goal>enforce</goal>-->
+            <!--                        </goals>-->
+            <!--                        <phase>validate</phase>-->
+            <!--                    </execution>-->
+            <!--                </executions>-->
+            <!--            </plugin>-->
         </plugins>
```

Comment:
Is there a reason for removing all the excludes? As noted too I'd delete instead of comment out.

Reply:
I currently disable the `enforce-plugin`, I'll re-enable it at last and adjust all the exclusions.